pkg/restore/restore.go

```diff
@@ -814,6 +823,35 @@ func hasControllerOwner(refs []metav1.OwnerReference) bool {
 	return false
 }
 
+// isComplete returns whether or not an object is considered completed
+// Used to identify whether or not an object should be restored. Only Jobs or Pods are considered
+func isCompleted(obj *unstructured.Unstructured, groupResource schema.GroupResource) (bool, error) {
+	podsGroupResource := schema.GroupResource{Group: "", Resource: "pods"}
```

Comment:
we have some similar ones defined in `pkg/backup/item_backupper.go`, for different resources. WDYT about moving these into a shared pkg and exporting them?

Reply:
Taking a look, I'm not seeing independent helper funcs in `item_backupper.go`. They're all methods.

I see 4 in `backup.go` - were those perhaps the ones you were referring to?

Reply:
sorry, I meant the schema.GroupResource definitions

Reply:
That makes more sense :) I considered doing this, but wanted to keep the change here relatively small. By updating all other uses of it, the change is touching several backup package files that aren't otherwise relevant to this change.

Where would you put them when they are moved? I was thinking of a new file in the `pkg/util/kube` package, but I'm not sure that's the best place.

Reply:
Yeah, that's probably where I'd put them. Can just put a TODO in the code and circle back.

Reply:
Comment added and the other changes have been addressed.